modules/apps/transfer/keeper/migrations.go

```diff
@@ -81,6 +81,8 @@ func (m Migrator) MigrateTotalEscrowForDenom(ctx sdk.Context) error {
 		m.keeper.SetTotalEscrowForDenom(ctx, totalEscrow)
 	}
 
+	logger := m.keeper.Logger(ctx)
+	logger.Info("successfully set total escrow for %d denominations", totalEscrowed.Len())
```

Comment:
do we also want to log the `totalEscrow` amount?

Reply:
Not sure if that would add much value to the reader? I just wanted to have a simple message, since Stride run the migration and noticed that there was no indication in the logs of whether the migration succeeded or not.